src/_data/plugins/readme.md

```diff
@@ -0,0 +1,11 @@
+# Eleventy Community Plugins
+
+If you would like to add a plugin to [this](https://www.11ty.dev/docs/plugins/#community-contributed-plugins) list, create a PR adding a JSON file with the plugins name as the filename and the following content:
+
+```json
+{
+  "npm": "the npm package name of your plugin",
+  "author": "Your github username",
```

Comment:
Actually, the `author` is a _Twitter_ handle currently.

Reply:
Ahhh good to know, will fix